test/KernelWrap/kernel_wrap.ll

```diff
@@ -0,0 +1,65 @@
+; RUN: clspv-opt %s -o %t.ll --passes=wrap-kernel
+; RUN: FileCheck %s < %t.ll
+; RUN: spirv-val %t.spv
```

Comment:
This line should not be here. You are not producing any SPIR-V code, thus no validation is needed.